js/components/main.js

```diff
@@ -511,6 +511,10 @@ class Main extends ImmutableComponent {
 
     const onWindowMove = debounce(function (event) {
       const position = event.sender.getPosition()
+      // Windows may be placed off-screen
+      // See https://github.com/brave/browser-laptop/issues/3558
+      position[0] = Math.max(0, position[0])
```

Comment:
oh right, it's for all windows, but depending on the configuration you could still have gaps that fall outside any window without being negative. I think testing for onscreen points would still be safer, but for now we could use 0 in the Y axis and use window width to ensure a fixed number of > 0 pixels for the X axis